index.html

```diff
@@ -236,6 +251,14 @@ <h2>Other</h2>
     {"name": "Python Valencia 2023", "filename": "Python Valencia 2023.webp", "home": "https://python-valencia.es/", "mastodon": "https://masto.es/@python_vlc"},
     {"name": "Swiss Python Summit", "filename": "Swiss Python Summit.svg", "home": "https://www.python-summit.ch/", "twitter": "pythonsummit"},
   ];
+  const north_america_logos=[
+    {"name": "Python Mexico", "filename": "MX-PythonMexico.png", "home": "http://python.org.mx/"},
+    {"name": "México - PyMTY", "filename": "MX-pymty.png", "home": "https://social.org.mx/@pymty"},
```

Comment:
This site is mainly for variations on the official logo, plus some inspired by the two snakes. I think this Monterrey one is a bit outside the scope, so let's skip it here.

```suggestion
```

And please could you remove the `img/MX-pymty.png` file from the PR?